### PR TITLE
Update projects to .NET 9

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
     "name": "Development Jellyfin Server",
-    "image":"mcr.microsoft.com/devcontainers/dotnet:8.0-jammy",
+    "image":"mcr.microsoft.com/devcontainers/dotnet:9.0-jammy",
```
Comment (Gauvino) [inline .devcontainer/devcontainer.json:3]: Same as above

File: Jellyfin.Server/Helpers/StartupHelpers.cs [hunk 2]
```diff
@@ -292,13 +292,5 @@ public static void PerformStaticInitialization()
...
         // Ref: https://docs.microsoft.com/en-us/dotnet/api/system.text.codepagesencodingprovider.instance?view=netcore-3.0#remarks
         Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
-
-        // Increase the max http request limit
-        // The default connection limit is 10 for ASP.NET hosted applications and 2 for all others.
-        ServicePointManager.DefaultConnectionLimit = Math.Max(96, ServicePointManager.DefaultConnectionLimit);
-
-        // Disable the "Expect: 100-Continue" header by default
-        // http://stackoverflow.com/questions/566437/http-post-returns-the-error-417-expectation-failed-c
-        ServicePointManager.Expect100Continue = false;
```
Comment (crobibero) [inline Jellyfin.Server/Helpers/StartupHelpers.cs:302]: Are these just no longer needed?

**Review by JPVenson — APPROVED** [review]

File: Jellyfin.Server/Helpers/StartupHelpers.cs (hunk 2 above)
Comment (Bond-009) [reply Jellyfin.Server/Helpers/StartupHelpers.cs:302]: > In modern .NET, HttpClient does not take into account any configurations set on ServicePointManager.

Not sure how long this has been the case

**Review by Shadowghost — APPROVED** [review]

**Review by crobibero — APPROVED** [review]

**Comment by zent1n0** [comment]
Latest build still uses dotnet 8.0. Is this by design or just an accident that the commit is not merged into release?

**Comment by crobibero** [comment]
> Latest build still uses dotnet 8.0. Is this by design or just an accident that the commit is not merged into release?

10.10.x is net8.0, 10.11.x will be net9.0.

**Pull request merged**